products/antixlinux.md

```diff
@@ -0,0 +1,60 @@
+---
+permalink: /antix
+alternate_urls:
+-   /antixlinux
+-   /antix-linux
+title: antiX
+category: os
+releasePolicyLink: https://www.antixforum.com/forums/topic/when-is-end-of-support-for-stable-antix-versions-17-19/#post-26424
+iconSlug: mxlinux
+activeSupportColumn: true
+versionCommand: cat /etc/os-release
+releaseColumn: false
```

Comment:
```suggestion
releaseColumn: true
```

Considering there is important (and surprising) information below about 22 being the latest release in the 21 release cycle - we should show the latest release in each cycle.